tools/admin/wskadmin

```diff
@@ -105,6 +105,12 @@ def parseArgs():
     subcmd = subparser.add_parser('whois', help='identify user from an authorization key')
     subcmd.add_argument('authkey', help='the credentials to look up')
 
+    subcmd = subparser.add_parser('block', help='block a user')
+    subcmd.add_argument('subject', help='the user to block')
```

Comment:
on second thought - it may be something desirable _in the future_ (to block all subjects belonging to a namespace).